sci-libs/kissfft/kissfft-131.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="A Fast Fourier Transform (FFT) library that tries to Keep it Simple, Stupid"
+HOMEPAGE="https://github.com/mborgerding/kissfft"
+# QA checks don't like HOMEPAGE to be used in SRC_URI it seems
+BASE_URI="https://github.com/mborgerding/kissfft"
```

Comment:
why is this done?
all ebuilds just write it out explicitly in EGIT_REPO_URI and SRC_URI, Its fine for a small repitition.

Reply:
yup, this looks wrong